cal/cal/static/js/app.js

```diff
@@ -111,29 +111,41 @@ analyticsApp.component('tags', {
 function TagsDetailCtrl($scope, $http) {
   var tagUrl = '/v1/tags/' + this.tagId + '/events';
   var eventweek = '/v1/tags/' + this.tagId + '/eventWeek';
+  var query_timezone = moment.tz.guess();
   $scope.tagDetails = [];
   $scope.tagEvents = [];
+  $scope.tagHours = this.tagHours;
```

Comment:
The problem isn't referring it in the html file, but changing the instances in app.js to be this.tagDetails. It gives an error. I'm just going to merge so Cindy can work on it, and I'll work on scope soup in my next PR